roles/servicetelemetry/templates/manifest_smartgateway_events.j2

```diff
@@ -1,21 +1,35 @@
-apiVersion: smartgateway.infra.watch/v2alpha1
+apiVersion: smartgateway.infra.watch/v2
 kind: SmartGateway
 metadata:
   name: '{{ this_smartgateway }}'
   namespace: '{{ meta.namespace }}'
 spec:
+{% if this_collector.debug_enabled is defined and this_collector.debug_enabled %}
+  logLevel: "debug"
+{% else %}
+  logLevel: "info"
+{% endif %}
+  handleErrors: true
   size: {{ smartgateway_deployment_size }}
-  amqpUrl: '{{ meta.name }}-interconnect.{{ meta.namespace }}.svc.cluster.local:5672/{{ this_collector.subscription_address }}'
-  amqpDataSource: '{{ this_collector.collector_type }}'
-  serviceType: 'events'
-  debug: {{ this_collector.debug_enabled | default(false) }}
-  elasticUrl: 'https://elasticsearch-es-http.{{ meta.namespace }}.svc.cluster.local:9200'
-  useBasicAuth: true
-  elasticUser: '{{ elastic_user | default('elastic') }}'
-  elasticPass: '{{ elastic_pass | default('') }}'
-  useTls: true
-  tlsClientCert: /config/certs/tls.crt
-  tlsClientKey: /config/certs/tls.key
-  tlsCaCert: /config/certs/ca.crt
-  tlsServerName: 'elasticsearch-es-http.{{ meta.namespace }}.svc.cluster.local'
-  resetIndex: false
+  applications:
+  - config: |
+      hostURL: https://elasticsearch-es-http.{{ meta.namespace }}.svc.cluster.local:9200
+      useTLS: true
```

Comment:
I spent probably an hour trying to figure out why this wasn't working. Needed `URL` and `TLS` to be full caps...